app/assets/stylesheets/custom.scss

```diff
@@ -67,3 +67,15 @@
 .col-md-6 {
   width: 50%;
 }
+
+.prop_listings {
+  text-align: center;
+}
+
+.marquee-listing-image {
+  display: block;
+  -webkit-margin-before: 1em;
```

Comment:
Properties should be ordered display, -webkit-margin-after, -webkit-margin-before, -webkit-margin-end, -webkit-margin-start